web/pgadmin/authenticate/oauth2.py

```diff
@@ -151,6 +151,22 @@ def login(self, form):
                 current_app.logger.exception(error_msg)
                 return False, gettext(error_msg)
 
+        if self.oauth2_config[self.oauth2_current_client][
```

Comment:
If OAUTH2_ADDITIONAL_CLAIMS is not specified (even as None) in the config, this code will return an error. Make sure it exists in the array, and proceed if it exists.

Add test cases for this config. You can modify the test_oauth2_with_mocking.py file to add the test cases.

Reply:
will do

Reply:
added validation for success check on OAUTH2_ADDITIONAL_CLAIMS, but I don't know how to assert a failure and I could not find examples. If this is mandatory, can you help me here?